examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -4,7 +4,9 @@ using LinearAlgebra: norm_sqr
 using ClimaCore:
     Geometry, Domains, Meshes, Topologies, Operators, Spaces, Fields
 
+include("../implicit_solver_debugging_tools.jl")
 include("../ordinary_diff_eq_bug_fixes.jl")
+include("../common_spaces.jl")
 include("schur_complement_W.jl")
 
 const R = 6.371229e6    # Earth's radius
```

Comment:
Can we move this to each of the experiment files, e.g., `baroclinic_wave_rhoe.jl`? There will be chances that people want to run a different planet radius (not sure whether in ClimaCore or not but better to give this flexibility.

Reply:
Common spaces does not specify the planet radius `R`. That is included in `baroclinic_wave_utilites.jl`. Any test that does not include `baroclinic_wave_utilities.jl` must specify its own radius.

Reply:
I only added `common_spaces.jl` so that I could specify the 2D/3D box test cases using the same format as the sphere test cases (both can just specify `space`, rather than `velem`, `helem`, etc.)